tests/phpunit/payment-methods/class-wc-stripe-upe-payment-method-test.php

```
@@ -1142,4 +1208,4 @@ function ( $payment_method_class ) {
 
 		return $test_cases;
 	}
-}
+}
```

PHPMD (2.15.0)

[warning] The class WC_Stripe_UPE_Payment_Method_Test has 1206 lines of code. Current threshold is 1000. Avoid really long classes. (undefined)

(ExcessiveClassLength)